identity-service/src/routes/notifications.js

```diff
@@ -377,11 +435,12 @@ module.exports = function (app) {
         ],
         include: [{
           model: models.SolanaNotificationAction,
-          required: true,
+          required: false,
```

Comment:
just curious what this changes

Reply:
prior to my inclusion of the metadata column, we would store metadata like things with this NotificationAction object. If required = true, and we don't have a corresponding NotificationAction for a notification, this query won't return the notif at all